apps/extension/src/ui/domains/Mnemonic/MnemonicUnlock.tsx

```diff
@@ -90,16 +89,15 @@ const BaseMnemonicUnlock: FC<MnemonicUnlockProps> = ({
     if (!mnemonic) setFocus("password")
   }, [mnemonic, setFocus])
 
-  useEffect(() => {
-    return () => {
-      setValue("password", "")
-    }
-  }, [setValue])
-
   return mnemonic ? (
-    <>{children}</>
+    <div className={`${DEFAULT_WIDTH} ${mnemonic.split(" ").length > 12 && LARGE_WIDTH}`}>
```

Comment:
doing a test and `&&` could pollute the output class attribute with classes such as "false" or "undefined" in some cases, might want to leverage classNames() helper to prevent that

Reply:
Good one